enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+
+### User Stories
+
+* As a fleet administrator, I want to install a homogeneous log collection and forwarding on any set of RHACM managed OCP clusters.
+* As a fleet administrator, I want to install a homogeneous trace collection and forwarding on any set of RHACM managed OCP clusters.
+* As a fleet administrator, I want to centrally control authentication credentials for the target log forwarding outputs.
+* As a fleet administrator, I want to centrally control authentication credentials for the target trace exporters.
+
+### Goals
+
+* Provide an optional RHACM addon to control collection and forwarding of logs and traces on managed OCP clusters.
+* Enable control of authentication and authorization of storage endpoints from the hub cluster.
+* Enable TLS Certificate management local on the hub cluster for managed cluster TLS client certificates.
+
+### Non-Goals
+
+* Provide end-to-end experience to collect, forward, store and visualize logs and traces on the same RHACM fleet.
+* Share MCO's ingestion and querying capabilities for logs and traces collected by MCOA.
+
+## Proposal
+
+The following sections describe in detail the required resources as well as the workflow to enable collection and forwarding of logs and traces on an RHACM managed fleet of OCP clusters.
+
+### Workflow Description
+
+The workflow implemented in this proposal enables fleet-wide collection and forwarding of logs and traces as follows:
+
+1. When the fleet administrator either enables either Logs and/or Traces in the `MultiClusterObservability` resource, then MCOA is deployed by MCO.
+2. The fleet administrator creates a default `ClusterLogForwarder` stanza in the `open-cluster-management` namespace that describes the list of log forwarding outputs. This stanza will then be used as a template by MCOA when generating the `ClusterLogForwarder` instance per managed cluster.
+3. The fleet administrator creates a default `OpenTelemetryCollector` stanza in the `open-cluster-management` namespace that describes the list of trace receivers, processors, connectors and exporters. This stanza will then be used as a template by MCOA when generating the `OpenTelemetryCollector` instance per managed cluster.
+4. For each managed cluster the MCOA or the fleet administrator provides on the managed cluster namespace additional configuration resources:
+   1. For both Logs and Traces a `ConfigMap` with the authentication methods that should be used for each Log Output/ Trace Exporter. 
+   2. Per Log Output / Trace Exporter `ConfigMap`: For each output a resource holding output specific configuration (See [ClusterLogForwarder Type: OutputTypeSpec](ocp-clusterlogforward-outputtypespec), [OpenTelemetry Collector Authentication](opentelemetry-collector-auth))
+   3. Each resource created should be annotated with a special label `mcoa.openshift.io/signal` that should be set to the resource that should use that resource.
```

Comment:
This sentence is unclear to me.  `should be set to the resource that should use that resource` is confusing and I don't really understand what purpose the label should serve.

Reply:
This was part of the old design this was removed in the new design